src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getDefaultFigure() {
+        return new Circle(10, Color.WHITE);
+    }
+
+    public Figure getRandomFigure() {
+        int number = new Random().nextInt(5);

```

Comment:
create separate method for getting random side

5 is a magic number - create constant